Mathlib/Analysis/Calculus/TangentCone.lean

```diff
@@ -207,6 +208,125 @@ theorem mem_tangentCone_of_segment_subset {s : Set G} {x y : G} (h : segment ℝ
     y - x ∈ tangentConeAt ℝ s x :=
   mem_tangentCone_of_openSegment_subset ((openSegment_subset_segment ℝ x y).trans h)
 
+/-- The tangent cone at a non-isolated point contains `0`. -/
+theorem zero_mem_tangentCone {s : Set E} {x : E} (hx : (𝓝[s \ {x}] x).NeBot) :
+    0 ∈ tangentConeAt 𝕜 s x := by
```

Comment:
```suggestion
theorem zero_mem_tangentConeAt {s : Set E} {x : E} (hx : (𝓝[s \ {x}] x).NeBot) :
    0 ∈ tangentConeAt 𝕜 s x := by
```

Reply:
All other lemmas about the tangent cone use `tangentCone` and not `tangentConeAt`. I agree it's strange, but I didn't want to depart from this convention in one single lemma. If we want to do the change, it should rather be in another single renaming PR.